CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 ## 1.79.2
 
-* `selector.unify()` now preserves the original order of pseudo-classes and
-  pseudo-elements within individual selectors when combining them.
+* Changes to how `selector.unify()` and `@extend` combine selectors:
+
+    * The relative order of pseudo-classes (like `:hover`) and pseudo-elements
+      (like `::before`) within each original selector is now preserved when
+      they're combined.
+
+    * Pseudo selectors are now consistently placed at the end of the combined
+      selector, regardless of which selector they came from. Previously, this
+      reordering only applied to pseudo-selectors in the second selector.
+
+    * Prioritizes components from the first selector in the output.
```

Comment:
"Prioritizes" in what sense?

Reply:
Ah, interesting... So, if you notice, I changed the order of which selector gets unified into the other. It used to be that the first selector was unified into the other in `unifyCompound`. For example:

```
print(unifyCompound(
  CompoundSelector.parse('.x'),
  CompoundSelector.parse('.y'),
));
// > .y.x
```
And I didn't like that 😅 

But apparently `selector.unify()` doesn't have this issue 🤔 [[playground](https://sass-lang.com/playground/#eJwzNHQoLU5VUCpOLC62Kk7NSU0uyS9SsuZySElNKk1XgInoleZlplVqqOtVqOsoqOtVWlkVJxfl5+QkJRZZlaUWlWQmJ+aoawIA/RkaYQ==)]

Since that didn't change behavior, I deleted this line.